src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -375,14 +400,17 @@ public void reportItemsInTransitSortedByCheckInServicePoint() {
     createRequest(smallAngryPlanet, steve, firstServicePointId, requestDate1, requestExpirationDate1);
     createRequest(nod, rebecca, secondServicePointId, requestDate2, requestExpirationDate2);
 
+    mockClockManagerToReturnFixedDateTime(checkInDate3);
     loansFixture.checkInByBarcode(new CheckInByBarcodeRequestBuilder()
       .forItem(smallAngryPlanetWithFourthCheckInServicePoint)
       .on(checkInDate3)
       .at(fourthServicePointId));
+    mockClockManagerToReturnFixedDateTime(checkInDate2);
     loansFixture.checkInByBarcode(new CheckInByBarcodeRequestBuilder()
       .forItem(nod)
       .on(checkInDate2)
       .at(firstServicePointId));
+    mockClockManagerToReturnFixedDateTime(checkInDate1);
```

Comment:
Personally, I'd put some white space in between these separate check ins, to make it clearer that they are separate steps

Reply:
Sure.